### ISSUE TITLE
running scripts before task start

### ISSUE THREAD

**Issue by ekimia** [aws/containers-roadmap#366]
I have volumes id like to delete using docker rm volume VOLUME_NAME but dont see a way to do it when the agent receives a new task. Adding the ability to execute a script before a task executes, or specifying the command to run on the instance beforehand would be awesome.

**Comment by coultn** [comment]
FYI for anyone watching this issue:  with [container dependencies](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ContainerDependency.html), you can specify a container that must finish before others in the task can start.  This allows you to run scripts inside that container that run before the rest of the task's containers run.
